src/App.tsx

```diff
@@ -20,6 +20,7 @@ import JudeHome from 'screens/JudeHome';
 import KenHome from 'screens/KenHome';
 import MatthewHome from 'screens/MatthewHome';
 import VeronicaHome from 'screens/VeronicaHome';
+import SemaHome from 'screens/SemaHome';
```

Comment:
i like this, however please order the imports by alphabetical order (i should have a lint rule for that)